test_regress/t/t_vpi_dump.out.iv

```diff
@@ -0,0 +1,252 @@
+t (vpiModule) t
```

Comment:
Rename e.g. t_vpi_dump__iv.out.  I prefer .out suffixes for all golden text files.